tests/wallet_main_screen/wallet: plus button/test_plus_button_manage_account_from_seed_phrase.py

```diff
@@ -83,3 +85,35 @@ def test_plus_button_manage_account_from_seed_phrase(main_screen: MainWindow, us
     with step('Verify that the account is not displayed in accounts list'):
         assert driver.waitFor(lambda: new_name not in [account.name for account in wallet.left_panel.accounts], 10000), \
             f'Account with {new_name} is still displayed even it should not be'
+
+
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/736371',
+                 'Try re-importing seed phrase when adding account')
```

Comment:
Maybe

```suggestion
                 'Can't import the same seed phrase in wallet'
```
? No strong opinion, but perhaps it gives better idea what to expect?

Reply:
Done